Misc/NEWS.d/next/Core and Builtins/2018-12-22-22-19-51.bpo-35560.9vMWSP.rst

```diff
@@ -0,0 +1,3 @@
+Removed an incorrect assert statement expecting `min_width`  >= 0 in
+`_PyUnicode_InsertThousandsGrouping` that caused segfault in :func:`format`
+in debug builds for certain cases. Patch by Karthikeyan Singaravelan.
```

Comment:
What are certain cases? This is a concrete case: floating point formatting with zero padding and small width.

Reply:
Is the below content okay? It will be helpful if you have any in mind so that I can use it since I don't know how to explain it without the internals in the news entry.

```rst
Removed an incorrect assert statement that caused segfault in :func:`format`
in debug builds for floating point formatting with zero padding and small width. 
Patch by Karthikeyan Singaravelan.
```

Reply:
It LGTM, but native speakers can suggest better wording.